scripts/cli-e2e-test.js

```diff
@@ -15,11 +15,18 @@
  */
 
 const { resolve: resolvePath } = require('path');
-const childProcess = require('child_process');
-const { spawn } = childProcess;
 const Browser = require('zombie');
 
-const EXPECTED_LOAD_ERRORS = /ECONNREFUSED|ECONNRESET|did not get to load all resources/;
+const {
+  spawnPiped,
+  handleError,
+  waitForPageWithText,
+  waitForExit,
+  print,
+} = require('./helpers');
```

Comment:
If we're breaking it up into multiple files I think it would be better to either move it into a folder, or probably move it into packages/cli. Can do that later though

Reply:
👍